src/main/java/seedu/address/logic/command/FindCommand.java

```diff
@@ -52,8 +52,8 @@ public FindCommand(NameContainsKeywordsPredicate predicate, int travelPlanObject
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
-        if (!model.isDirectoryTypeTravelPlan() && this.travelPlanObjectType != 0) { //wishlist
-            throw new CommandException(Messages.WRONG_DIRECTORY);
+        if (!model.isDirectoryTypeTravelPlan()) { //wishlist
```

Comment:
I think need to check if the travelPlanObjectType is not Activity before throwing exception? Otherwise cannot find inside Wishlist.

Reply:
Oya, forgot abt that. Thanks!